CHANGELOG.md

```diff
@@ -1,5 +1,107 @@
 [//]: # (current developments)
 
+## 25.5.0 (2025-05-21)
+
+### Enhancements
+
+* Add the ability to define pre- and post-transaction hooks. (#14754)
+* Adds support for showing and manipulating settings registered via the settings plugin hook to `conda config`. (#13661 via #14510)
+* Raise `conda.exceptions.OfflineError` instead of generic `RuntimeError` when `--offline` is used for better error handling. (#14664 via #14665)
+* Add `context.default_activation_env` setting, which allows users to customize which environment should get auto activated on initialization (if `context.auto_activate` is enabled), as well as `conda activate` calls without arguments. Defaults to `base`. (#14666)
+* Provide `conda init --condabin`, which only adds the `$CONDA_PREFIX/condabin` directory to the PATH environment variable instead of installing a shell function. (#14703)
```

Comment:
```suggestion
* Add `conda init --condabin` to only add the `$CONDA_PREFIX/condabin` directory to the `PATH` environment variable instead of installing a shell function. (#14703)
```